src/app/pages/session-detail/session-detail.page.ts

```diff
@@ -135,8 +135,8 @@ export class SessionDetailPage implements OnInit {
 
   async fetchSessionDetails() { 
     var response = await this.sessionService.getSessionDetailsAPI(this.id);
-    this.sessionDatas = response.result;
```

Comment:
is this a needed change?

Reply:
yes else , sessionData is coming undefined some edge cases